packages/fcl/src/fcl.ts

```diff
@@ -1,10 +1,9 @@
-export * from "./shared-exports"
+export * from "@onflow/fcl-core"
```

Comment:
Can we namespace the exports from FCL core?  i.e. separate the ones that are supposed to be exported, shared exports between both packages & the ones that are only used internally.

That way way we avoid external consumers coupling with FCL internals (otherwise some exports like `getMutate`, etc. will be exported from `@onflow/fcl`

Reply:
good point!